src/Common/Commands.ScenarioTest/ServiceManagement/ServiceManagementTests.cs

```diff
@@ -21,19 +21,16 @@
 
 namespace Microsoft.WindowsAzure.Commands.ScenarioTest
 {
-    public partial class AzureVMTests
+    public partial class ServiceManagementTests
     {
         private EnvironmentSetupHelper helper = new EnvironmentSetupHelper();
 
-        #region Get-AzureVM Scenario Tests
-
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
```

Comment:
This test takes 151.519s which is too much. since you are working with these tests, could you take a look why it is soo long? Consider removing it from CheckIn category if it is too complex and there is no way to make it run shorter.